src/main/java/com/kodilla/ecommercee/entity/Item.java

```diff
@@ -0,0 +1,54 @@
+package com.kodilla.ecommercee.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+public class Item {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
```

Comment:
Ta kaskada wydaje się niepotrzebna. Raczej nie chcemy zapisywać koszyka poprzez zapisywanie jednego z Itemów wewnątrz niego. Będzie to raczej miało sens w druga stronę, czyli zapisujemy Cart z Itemami w środku